sklearn/neighbors/_nearest_centroid.py

```diff
@@ -68,6 +85,21 @@ class is the arithmetic mean, which minimizes the sum of squared L1 distances.
 
         .. versionadded:: 1.0
 
+    deviations_ : ndarray of shape (n_classes, n_features)
+        Deviation of each class using soft thresholding.
+
+        .. versionadded:: 1.4
+
+    within_class_std_dev_ : ndarray of shape (n_features,)
+        Within-class standard deviation with unshrunked centroids.
+
+        .. versionadded:: 1.4
+
+    class_priors_ : ndarray of shape (n_classes,)
```

Comment:
For consistency with other models, I think that we should name it `class_prior_` unfortunatelly even if the plural makes more sense.